compendium/ConfigurationAdmin/src/ConfigurationImpl.cpp

```diff
@@ -87,25 +87,38 @@ void ConfigurationImpl::Update(AnyMap newProperties)
   }
   std::lock_guard<std::mutex> lk{ configAdminMutex };
   if (configAdminImpl) {
-    configAdminImpl->NotifyConfigurationUpdated(pid);
+    std::shared_future<void> fut =
```

Comment:
Jeff: you can just "return" here
Patty: will check for more instances of this in other parts of the code